.bazelrc

```diff
@@ -3,6 +3,9 @@
 # Include this file in another .bazelrc with a line like:
 #   try-import submodules/dev-tools/.bazelrc
 
+# Print failed test logs to the console by default.
+build --test_output=errors
```

Comment:
@CodingCanuck should this be on `build` or on `test`?

Reply:
Good point! Sent you https://github.com/3rdparty/dev-tools/pull/32